deepchem/trans/transformers.py

```diff
@@ -1611,6 +1610,7 @@ def realize(self, similarity: np.ndarray, y: np.ndarray,
       n_samples * np.array of size (2*K,)
       each array includes K similarity values and corresponding labels
     """
+    import tensorflow as tf
```

Comment:
We should figure out why transformers are using TensorFlow in the first place.  So far as I know, all transformers should be operating just on numpy arrays.